docs/application/toc_all.md

```diff
@@ -989,9 +989,9 @@
 #### [Tracepoints](/application/native/guides/performance/tracepoints.md)
 
 ### Watchface Complication
-#### [Overview](/application/native/guides/compliations/overview.md)
-#### [Watchface Complication](/application/native/guides/compliations/watchface-complication.md)
-#### [Complication Provider](/application/native/guides/compliations/watchface-complication-provider.md)
+#### [Overview](/application/native/guides/complications/overview.md)
+#### [Watchface Complication](/application/native/guides/complications/watchface-complication.md)
+#### [Complication Provider](/application/native/guides/complications/watchface-complication-provider.md)
```

Comment:
folder name is **complications**.
Please change the folder name too.

Reply:
@journey2w 
The folder name seems already complications.
Could you check it again?
https://github.com/Samsung/tizen-docs/tree/master/docs/application/native/guides/complications

Reply:
You are right. I might see staging server.